### DISCUSSION TITLE
https://www.yunyitang.me/vercel-host-umami-analytics-hexo/

### DISCUSSION THREAD

**Discussion by giscus** [yunyit/yunyit.github.io#176] [Comments]
# https://www.yunyitang.me/vercel-host-umami-analytics-hexo/

一般来说，写博客的都喜欢在页面上加上一个访问计数器，来满足虚荣心显示某篇文章或者整个站点的访问量。这种需求在 WordPress 等动态博客上都是比较容易满足的，安装个插件即可（辣鸡 Ghost 除外），但是对于小部分静态博客来说就比较头疼了。 目前来看，互联网上的静态博客访问计数器解决方案大致有这么几种：  使用“不蒜子”访问计数服务； 各平台统计服务。

https://www.yunyitang.me/vercel-host-umami-analytics-hexo/

**Comment by wumingblog** [comment]
如果需求不大的话vercel有自带的统计很好用:)

**Reply by yunyit** [reply]
好呢，我去看看！

**Comment by lkcdm** [comment]
自打我的umami又炸了我也彻底摆烂了

**Reply by yunyit** [reply]
umami更新但我也摆烂了（
